Lookuptable/README.md

```diff
@@ -0,0 +1,57 @@
+## Commcare lookuptable Test Script
+
+Lookup table tests ensure that the important features are working and there are no showstoppers in the build deployed to environments.\
+The automated tests comprises of [these Lookuptable functionality.](https://docs.google.com/spreadsheets/d/1-Fqax67kLz8dIJE5UcM1iGeWCV9mAm5OH6lZwb290cQ/edit#gid=0)
```

Comment:
Can we please link the master lookup table test plan here instead: https://docs.google.com/spreadsheets/d/1aTBv4zGeVuNTdnCyuleZbbpqOhPEGl0f78rcKAGWQHQ/edit#gid=1360466241?

Reply:
changes made . Here is the commit id for reference: 99b00a9f008a562cec8a5a54baac05797b07c2c2